tests/js/HappinessLintBearTest.py

```diff
@@ -17,3 +24,18 @@
 HappinessLintBearTest = verify_local_bear(HappinessLintBear,
                                           valid_files=(good_file,),
                                           invalid_files=(bad_file,))
+
+
+@generate_skip_decorator(HappinessLintBear)
+class HappinessLintBearConfigTest(LocalBearTestHelper):
+
+    Value_Error = ('"use_spaces=True" is incompatible with HappinessLintBear, '
+                   'set it to false.')
+
+    def test_validate_use_spaces(self):
+        section = Section('name')
+        section.append(Setting('use_spaces', True))
+        bear = HappinessLintBear(section, Queue())
+
+        with self.assertRaisesRegex(AssertionError, self.Value_Error):
+            self.check_validity(bear, [], good_file)
```

Comment:
still please don't use `check_validity`, but try to run the bear manually (probably there's something like `execute_bear`)

Reply:
nope no such attribute.

Reply:
ohh sry its not an attribute my bad

Reply:
i am not able to get exactly what execute_bear does

Reply:
![image](https://user-images.githubusercontent.com/25884863/33489003-945ac288-d6d8-11e7-99e8-334eef595581.png)
m i going in the right direction?

Reply:
No, the `with self.assertRaisesRegex` part is totally okay, but I want you to execute the bear without using `check_validity`, as you don't use this function with the intent to check its validity, but just execute the bear. Just read up the code of `check_validity` itself to see how the bear is run.

Reply:
yes i read that and as far as i came to know it checks if the bear is valid or not using a proper `goodFile`.

Reply:
yes and you shall use the part that is just running the bear from that code, copy the parts you need :3

Reply:
@Makman2 i did made the tests run well but dont know why the circle-ci is failing

Reply:
![image](https://user-images.githubusercontent.com/25884863/33518488-ba2f6912-d7bb-11e7-9d2a-d6e95978c8d8.png)